Multiplatform/Collections/Playlists/PlaylistAddSheet.swift

```diff
@@ -14,8 +14,11 @@ struct PlaylistAddSheet: View {
     
     let track: Track
     
+    @State private var serverVersion: String?
```

Comment:
This can be removed

Reply:
Resolved in 712e3723c0e3dd353082cbd1624092ce1a94dff9